lib/engine/game/g_1840/step/dividend.rb

```diff
@@ -37,7 +37,12 @@ def process_dividend(action)
             amount = action.amount || 0
             payout = corp_dividend_options(entity, amount)[kind]
 
-            raise GameError, "Amount must be multiples of #{variable_input_step}" if amount % variable_input_step != 0
+            if (amount > variable_max(entity)) || amount.negative?
+              raise GameError,
+                    "Dividend cannot be larger than total revenue of #{@game.format_currency(variable_max(entity))} "\
+                    "or smaller than #{@game.format_currency(0)}"
```

Comment:
Nit: Do we really need to explain that the dividend can't be negative? :)

If we do, how about shortening it to something like "Dividend cannot be negative or larger than total revenue ($total)"

Reply:
I don't have a strong preference, the user is only running into this error if they've done something wonky so it seems OK